client/components/ReviewRating/ReviewList/reviews.js

```diff
@@ -0,0 +1,36 @@
+const reviews = [
```

Comment:
This is mock data right? Will it be removed at some point?

Reply:
Yes, it can be remove.